### ISSUE TITLE
Occupational disability

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6774]
**Preferred term label:**
Occupational disability

**Synonyms**

**Definition (free text, please give PubMed ID)**
This is a general term that denotes a reduced ability to perform the work that one performed prior to an illness, and may be related to pain, cognitive dysfunction, fatigue or other physical disabilities.

Comment
This term does not intend to include or exclude legal definitions of occupational disability. It is included in the HPO because inability to work is a frequent manifestation of disease comparable in some ways to impairment of activities of daily living.

**Parent term (use hpo.jax.org/app)**
Constitutional symptom HP:0025142

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
aadded
